apps/sertra/pages/create.tsx

```diff
@@ -39,37 +71,43 @@ const SubmitServer: NextPage<{ swrFallback: { [key: string]: unknown } }> = ({
             <input
               className={styles.input}
               id={"serverName"}
-              name={"serverName"}
               type={"text"}
+              {...register("serverName", { required: true })}
             />
+            {/* Add styling here */}
```

Comment:
Should this be handled now? Or at least add TODO to make these easier to find in the future.

Reply:
Yeah, I'm not sure what is the correct styling for this. Just a red border warning or an actual message 🤷 I'll add a ticket about this.

Reply:
> I'm not sure what is the correct styling for this.
Just have the comment start with `TODO:`, doesn't matter what the wrapping syntax is